sklearn/pipeline.py

```diff
@@ -410,6 +410,7 @@ def predict(self, X, **predict_params):
             or return_cov, uncertainties that are generated by the
             transformations in the pipeline are not propagated to the
             final estimator.
+            ..versionadded:: 0.20
```

Comment:
@adrinjalali I guess it was added because of this piece in the changelog:

```- |Feature| The ``predict`` method of :class:`pipeline.Pipeline` now passes
  keyword arguments on to the pipeline's last estimator, enabling the use of
  parameters such as ``return_std`` in a pipeline with caution.
  :issue:`9304` by :user:`Breno Freitas <brenolf>`.```

Should I change `versionadded` to `versionchanged`?

Reply:
yeah, `versionchanged` would be appropriate here.